### PR TITLE
Integrate SegmentStore Repair method with repair service

### PR DESCRIPTION
**Pull request by cam-a**
The storeConfig fields are currently defaulted to work with captplanet defaults. Repair method has been tested, but the service as a whole still needs testing

### CODE REVIEW

File: cmd/captplanet/run.go [hunk 1]
```diff
@@ -131,7 +131,7 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 			o,
 			// TODO(coyle): re-enable the checker and repairer after we determine why it is panicing
 			// runCfg.Satellite.Checker,
-			// runCfg.Satellite.Repairer,
+			runCfg.Satellite.Repairer,
```
Comment (kaloyan-raev) [inline cmd/captplanet/run.go]: Should we update the TODO comment above?

File: pkg/datarepair/repairer/config.go [hunk 2]
```diff
@@ -19,6 +27,15 @@ type Config struct {
 	QueueAddress string        `help:"data repair queue address" default:"redis://127.0.0.1:6378?db=1&password=abc123"`
 	MaxRepair    int           `help:"maximum segments that can be repaired concurrently" default:"100"`
 	Interval     time.Duration `help:"how frequently checker should audit segments" default:"3600s"`
+	storeConfig
+}
+
+type storeConfig struct {
```
Comment (kaloyan-raev) [inline pkg/datarepair/repairer/config.go]: Why is this a new struct? Can't we reuse the `ClientConfig`?

Comment (cam-a) [reply pkg/datarepair/repairer/config.go]: Done

File: cmd/captplanet/run.go (hunk 1 above)
Comment (kaloyan-raev) [reply cmd/captplanet/run.go]: Oh. I simply meant we can update the comment to reflect that the repairer is now re-enabled, i.e. the comment should mentioned just the checker. The repairer is not panicking any more, right?

Comment (cam-a) [reply cmd/captplanet/run.go]: done

**Pull request merged**